recipes/wildmidi/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.files import export_conandata_patches, get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class WildmidiConan(ConanFile):
+    name = "wildmidi"
+    version = "0.4.5"
```

Comment:
```suggestion
```
Please, do not add version attribute, use conandata.yml instead, so we can re-use the same recipe for any future version.